### PR TITLE
Contact us page

### PR DESCRIPTION
**Pull request by full-profane**
image

### CODE REVIEW

File: volim.css [hunk 1]
```diff
@@ -320,89 +243,158 @@ body {
...
   display: block;
   width: 65%;
   height: 90%;
   position: absolute;
   right: 250px;
 }
+
+.contactus {
+  display: block;
+  height: 700px;
```
Comment (max-si-m) [inline volim.css]: please avoid `height` where is possible

File: volim.css [hunk 2]
```diff
@@ -320,89 +243,158 @@ body {
...
+.contactus-red {
+  display: block;
+  height: 80%;
+  background: #b52f3a;
+}
+
+.contactus-form {
+  display: inline-block;
+  width: 26%;
+  height: 70%;
```
Comment (max-si-m) [inline volim.css]: looks, redundant

File: volim.css [hunk 3]
```diff
@@ -320,89 +243,158 @@ body {
...
+  background: white;
+  margin-top: 60px;
+  margin-left: 170px;
+  padding-left: 50px;
```
Comment (max-si-m) [inline volim.css]: use one liner padding

File: volim.css [hunk 4]
```diff
@@ -320,89 +243,158 @@ body {
...
+  font-size: 16px;
+}
+
+.contactus-form__textarea {
+  width: 100%;
+  height: 70px;
+  border: 0;
+  margin-bottom: 30px;
+  border-bottom: 3px #928a8a solid;
+  font-size: 16px;
```
Comment (max-si-m) [inline volim.css:366]: does it different from base fonts?

File: volim.css [hunk 5]
```diff
@@ -320,89 +243,158 @@ body {
...
+  border: none;
+  color: white;
+  font-size: 18px;
+
+}
+
+.contactus-form__input:focus {
+  border: 1px red solid;
+  border-bottom: 3px #928a8a solid;
+  font-size: 12px;
```
Comment (max-si-m) [inline volim.css]: why font size is different from base state?

File: volim.css [hunk 6]
```diff
@@ -320,89 +243,158 @@ body {
...
+}
+.contactus-img {
+  display:block;
```
Comment (max-si-m) [inline volim.css:389]: space missed

File: volim.html [hunk 7]
```diff
@@ -6,7 +6,7 @@
   <link rel="stylesheet" type="text/css" href="volim.css">
 </head>
 <body>
-	<div class="red-container">
+	<div class="red-container ">
```
Comment (max-si-m) [inline volim.html]: redundant space

File: volim.html [hunk 8]
```diff
@@ -100,14 +100,14 @@
 			<p class="openhour-lesson__dance">Tab Dancing : 19:00 - 23.00</p>
 		</div>
 	</div>
-  <div class="shortcode">
-   <div class="shortcode-top">
-      <div class="shortcode-title">
-        <p class="shortcode-title__text">SHORTCODE</p>
-        <div class="shortcode-title__circle"></div>
+  <div class="shortcode ">
```
Comment (max-si-m) [inline volim.html]: redundant space

File: volim.html [hunk 9]
```diff
@@ -116,5 +116,25 @@
       </div>
    </div>
   </div>
+  <div class="contactus">
```
Comment (max-si-m) [inline volim.html]: change `countactus` -> `contact`

File: volim.css [hunk 10]
```diff
@@ -320,89 +243,158 @@ body {
...
+  border-bottom: 3px #928a8a solid;
+  font-size: 16px;
+  font-family: 'Arial';
+}
+
+.contactus-form__button {
+  display: block;
+  width: 50%;
+  height: 60px;
+  margin:  auto;
```
Comment (max-si-m) [inline volim.css]: extra space

File: volim.css [hunk 11]
```diff
@@ -320,89 +243,158 @@ body {
...
+  background: #b52f3a;
+  border: none;
+  color: white;
+  font-size: 18px;
+
```
Comment (max-si-m) [inline volim.css]: redundant empty line

File: volim.css [hunk 12]
```diff
@@ -320,89 +243,158 @@ body {
...
+}
+
+.contactus-form__button {
+  display: block;
+  width: 50%;
+  height: 60px;
+  margin:  auto;
+  background: #b52f3a;
+  border: none;
+  color: white;
```
Comment (max-si-m) [inline volim.css]: hex colors are preferred

**Review by max-si-m — CHANGES_REQUESTED** [review]

File: volim.css (hunk 4 above)
Comment (full-profane) [reply volim.css:366]: because the base font is smaller than i need

File: volim.css [hunk 13]
```diff
@@ -369,32 +367,39 @@ body {
...
+.contact-img__map {
   display: block;
   width: 70%;
-  height: 90%;
-  margin-top: 60px;
+  height: 92%;
+  margin-top: 80px;
+}
+
+footer {
```
Comment (max-si-m) [inline volim.css:401]: shouldn't be here :)

**Review by max-si-m — CHANGES_REQUESTED** [review]

File: volim.css (hunk 13 above)
Comment (full-profane) [reply volim.css:401]: fixed

File: volim.html [hunk 14]
```diff
@@ -116,5 +85,28 @@
...
+        <input  class="contact-form__input" type="email" name="email" placeholder="Email">
+        <textarea class="contact-form__textarea" type="text" name="message" placeholder="Message"></textarea>
+        <button class="contact-form__button">Send</button>
+      </div>
+      <div class="contact-img">
+        <img class="contact-img__map"c src="img/map_img.jpg">
+      </div>
+    </div>
+  </div>
+  <footer>
```
Comment (max-si-m) [inline volim.html]: not fixed :)

**Review by max-si-m — CHANGES_REQUESTED** [review]

File: volim.html (hunk 14 above)
Comment (full-profane) [reply volim.html]: image

**Review by max-si-m — APPROVED** [review]

**Pull request merged**